Dockerfile

```diff
@@ -18,6 +18,10 @@ RUN apt-get --no-install-recommends -y install \
     libmozjs185-dev \
     python
 
+# Install Nodejs
+RUN curl -sL https://deb.nodesource.com/setup | bash -	
```

Comment:
Hello everyone,
Nice to have feed back :D.
I found it in http://nodejs.org/download/ there is a link "Installing from package managers" so I took it . I thought it was a good think. 
Sorry if I did something wrong